app/qml/project/MMProjectHomePage.qml

```diff
@@ -0,0 +1,197 @@
+ *   This program is free software; you can redistribute it and/or modify  *
+ *   it under the terms of the GNU General Public License as published by  *
+ *   the Free Software Foundation; either version 2 of the License, or     *
+ *   (at your option) any later version.                                   *
+ *                                                                         *
+ ***************************************************************************/
+
+import QtQuick
+import lc 1.0
+
+import "../components"
+import "../misc"
+import "../inputs"
+import "."
+
+Item {
+  id: root
+
+  property string activeProjectId: ""
+  property alias list: projectlist
+
+  signal openProjectRequested( string projectFilePath )
+  signal showLocalChangesRequested( string projectId )
+
+  function refreshProjectsList() {
+    searchBar.deactivate()
+    projectlist.refreshProjectList( searchBar.text )
+  }
+
+  property int spacing: 10 * __dp
+
+  MMSearchInput {
+    id: searchBar
+
+    width: parent.width - 2 * root.padding
+    placeholderText: qsTr("Search for projects...")
+
+    onSearchTextChanged: function(text) {
+    }
+
+    anchors {
+      top: parent.top
+      left: parent.left
+      right: parent.right
+    }
+
+    allowTimer: true
+  }
+
+  MMWarningBubble {
+    id: warningBubble
+    visible: __merginApi.subscriptionInfo ? __merginApi.subscriptionInfo.actionRequired : false
+    width: root.width - 2 * root.hPadding
+    anchors {
+      top: searchBar.bottom
+      left: parent.left
+      right: parent.right
+      topMargin: root.spacing
+    }
+    title: qsTr("Your attention is required.")
+    description: qsTr("Please visit the %1Mergin dashboard%2.")
+    .arg("<a style=\"text-decoration: underline; color:" + __style.whiteColor + ";\" href='" + __inputHelp.merginDashboardLink + "'>")
+    .arg("</a>")
+
+    onClicked: Qt.openUrlExternally( __inputHelp.merginDashboardLink )
+  }
+
+  Column {
+    id: currentProjectColumn
+
+    width: parent.width
+    visible: root.activeProjectId !== ""
+
+    anchors {
+      top: warningBubble.visible ? warningBubble.bottom : searchBar.bottom
+      left: parent.left
+      right: parent.right
+      topMargin: 2 * root.spacing
+    }
+
+    Text {
+      width: parent.width
+      height: 31 * __dp
+      text: qsTr("Currently open")
+      font: __style.p6
+      color: __style.nightColor
+      wrapMode: Text.WordWrap
+      verticalAlignment: Text.AlignVCenter
+      elide: Text.ElideRight
+    }
+
+    MMLine {}
+
+    Item { width: 1; height: 20 * __dp }
+
+    Repeater {
+      width: parent.width
+      model: ProjectsProxyModel {
+        id: viewModel
+
+        projectSourceModel: ProjectsModel {
+          id: controllerModel
+
+          merginApi: __merginApi
+          localProjectsManager: __localProjectsManager
+          syncManager: __syncManager
+          modelType: ProjectsModel.LocalProjectsModel
+        }
+      }
+      delegate: MMProjectItem {
```

Comment:
Why you need repeater here ? it is just a single item , and all the data can be taken from model data via activeProjectId?